gems/aws-sdk-core/spec/aws/process_credentials_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../spec_helper'
+
```

Comment:
Test suites looks good, just checking does these covered all cli test suits for the process credentials as well?